pkg/networkpath/traceroute/runner.go

```diff
@@ -249,3 +303,28 @@ func getHostname(ipAddr string) string {
 	}
 	return currHost
 }
+
+func createGatewayLookup() (network.GatewayLookup, uint32) {
+	rootNs, err := rootNsLookup()
+	if err != nil {
+		log.Errorf("failed to create gateway lookup")
+	}
+	defer rootNs.Close()
+
+	nsIno, err := kernel.GetInoForNs(rootNs)
+	if err != nil {
+		log.Errorf("failed to create gateway lookup")
```

Comment:
1/ Looks like we are using the exact same error messages here and above. I think using a different msg can help troubleshooting later.

2/ Shall we suffix new error with original error message?
```
log.Errorf("failed to get ino for NS: %s", err)
```

same comment for the other error few lines above.